src/test/resources/HM22.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="HM for POM">
+    <test name="Test for logging in and creating playlist" preserve-order="false">
+        <classes>
+            <class name="tests.Homework22"/>
+            <methods>
+                <include name="createPlaylist"/>
+                <include name="addSongsPlaylist"/>
+                <include name="renamePlaylist"/>
+                <include name="deletePlaylist"/>
+            </methods>
```

Comment:
you got this error method is not allowed here because the scope of the class is closed within the class. But it should be closed after the methods are defined.
```
<class name="tests.Homework22">
            <methods>
                <include name="createPlaylist"/>
                <include name="addSongsPlaylist"/>
                <include name="renamePlaylist"/>
                <include name="deletePlaylist"/>
            </methods>
        </class>
```